### PR TITLE
Add 'portal' button to Odoo's login screen

### PR DESCRIPTION
**Pull request by MerlinGuillaume**
This commit adds the user 'portal' to the login page of Odoo

### CODE REVIEW

**Comment by MerlinGuillaume** [comment]
@hubvd Hello, could you please review this ?

File: userscripts/odoo-login.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 window.addEventListener('load', () => {
   const login = document.querySelector('button[type="submit"]')
-  for (let text of ['admin', 'demo']) {
+  for (let text of ['admin', 'demo', 'login']) {
```
Comment (hubvd) [inline userscripts/odoo-login.js]: It seems that you added the `login` user and not the `portal` user :monocle_face: 
```suggestion
  for (let text of ['admin', 'demo', 'portal']) {
```

**Review by hubvd — CHANGES_REQUESTED** [review]

File: userscripts/odoo-login.js (hunk 1 above)
Comment (MerlinGuillaume) [reply userscripts/odoo-login.js]: Sorry, it's fixed now

**Comment by hubvd** [comment]
@MerlinGuillaume Thank you for the fix :smiley:

**Pull request merged**